qiita_db/study.py

```diff
@@ -176,6 +182,28 @@ def get_by_status(cls, status):
         return {x[0] for x in conn_handler.execute_fetchall(sql, (status, ))}
 
     @classmethod
+    def get_info(cls, study_ids, info_cols=None):
+        if info_cols is None:
+            info_cols = [s for s in cls._info_cols]
+        else:
+            info_cols = deepcopy(info_cols)
```

Comment:
It doesn't seem like there's any need for this variable to be copied, am I missing something?